File: go/muse/util/test_runner/test_runner.go
package test_runner

import (
	"fmt"
	"os"
)

var itemIndex int

func Pick(fn func() bool) {
	if fn() {
		fmt.Printf("✓ Item [%d] PASSED\n", itemIndex)
	} else {
		fmt.Fprintf(os.Stderr, "✗ Item [%d] FAILED\n", itemIndex)
	}

	itemIndex++
}

File: python/muse/util/test_runner.py
import sys
from collections.abc import Callable

ItemIndex = 0


def pick(fn: Callable[[], bool]) -> None:
    global ItemIndex

    if fn():
        print("✓ Item [%d] PASSED" % ItemIndex)
    else:
        print("✗ Item [%d] FAILED" % ItemIndex, file=sys.stderr)

    ItemIndex += 1
